Frontend/ui-customer/src/Screens/CustomerUtils/ReportIssueChatScreen/Controller.purs

```diff
@@ -0,0 +1,475 @@
+    void $ pure $ startLottieProcess lottieAnimationConfig{ rawJson = "audio_upload_animation.json", lottieId = (getNewIDWithTag "audio_recording_done"), scaleType = "FIT_CENTER", speed = 1.0 }
+    void $ pure $ clearWaitingTimer state.props.timerId
+    case state.data.recordAudioState.recordedFile of
+      Just url -> do
+                  res <- runEffectFn1 saveAudioFile url
+                  _ <- runEffectFn3 uploadMultiPartData res (EndPoint.uploadFile "") "Audio"
+                  pure $  UpdateState state { data  { recordedAudioUrl = Just res}}
+      Nothing  -> do
+                  if state.data.recordAudioState.openAddAudioModel
+                  then do
+                    _ <- runEffectFn1 removeMediaPlayer ""
+                    pure $ UpdateState state { props { showRecordModel = false, showAudioModel = true }
+                                             , data  { addAudioState { stateChanged = true } } }
+                  else
+                    pure $ UpdateState state { data  { recordedAudioUrl = Nothing } }
+  ]
+
+eval (RecordAudioModelAction RecordAudioModel.OnClickRestart) state =
+  continueWithCmd state [do
+    _ <- runEffectFn1 removeMediaPlayer ""
+    pure $ UpdateState state { data { recordAudioState { isRecording = false, recordingDone = false } } }
+  ]
+
+eval (RecordAudioModelAction RecordAudioModel.OnClickClose) state =
+  if state.data.recordAudioState.openAddAudioModel
+  then
+    continueWithCmd state { props { showRecordModel = false }, data { recordAudioState { recordedFile = Nothing, recordingDone = false, isRecording = false, openAddAudioModel = false, isUploading = false, timer = "00 : 00" } } } [do
+      _ <- runEffectFn1 removeMediaPlayer ""
+      _ <- runEffectFn1 stopAudioRecording ""
+      _ <- pure $ clearWaitingTimer state.props.timerId
+      pure $ UpdateState state { props { showRecordModel = false, showAudioModel = true, isPopupModelOpen = true }
+                   , data { recordAudioState { recordedFile = Nothing, recordingDone = false, isRecording = false, openAddAudioModel = false, isUploading = false, timer = "00 : 00" }, addAudioState { audioFile = Nothing, stateChanged = true } } }
+    ]
+  else
+    continue state { data  { recordAudioState { recordedFile = Nothing } }
+                   , props { isPopupModelOpen = false, showRecordModel = false } }
+
+eval (RecordAudioModelAction RecordAudioModel.BackPressed) state = do
+  continueWithCmd state [do
+    if state.props.isPopupModelOpen
+    then pure (RecordAudioModelAction RecordAudioModel.OnClickClose)
+    else pure BackPressed
+  ]
+
+eval (UpdateRecordModelPlayer url) state = do
+  continueWithCmd state { data { recordAudioState { recordedFile = Just url } } } [do
+    _ <- addMediaFile (getNewIDWithTag "recordedAudioViewUniqueOne") url (getNewIDWithTag "actionButtonRecord") "ny_ic_play_recorded_audio" "ny_ic_pause_recorded_audio" "-1"
```

Comment:
@S-PRAVINKUMAR can we send a config to this function instead of sending 6 parameters, So that its easier to add parameters in future.

Reply:
@MercyQueen  can we do this refactoring in another PR? because there are almost 10 jbridge functions I added like this.